src/managing-workflow/tuning-component-settings.md

```diff
@@ -0,0 +1,228 @@
+edit `manifests/deis-controller-rc.yaml` and add the following under the `env` section:
+
+```
+env:
+  - name: REGISTRATION_MODE
+    value: "admin_only"
+```
+
+## Customizing the Builder
+
+The following environment variables are tunable for the [Builder][] component:
+
+Setting           | Description
+----------------- | ---------------------------------
+DEBUG             | Enable debug (default: false)
+
+## Customizing the Controller
+
+The following environment variables are tunable for the [Controller][] component:
+
+Setting                                         | Description
+----------------------------------------------- | ---------------------------------
+REGISTRATION_MODE                               | set registration to "enabled", "disabled", or "admin_only" (default: "enabled")
+GUNICORN_WORKERS                                | number of [gunicorn][] workers spawned to process requests (default: num_cpus * 2 + 1)
+RESERVED_NAMES                                  | a comma-separated list of names which applications cannot reserve for routing (default: "deis")
+SLUGRUNNER_IMAGE_NAME                           | the image used to run buildpack application slugs (default: "quay.io/deisci/slugrunner:canary")
+SLUG_BUILDER_IMAGE_PULL_POLICY                  | the kubernetes [image pull policy][pull-policy] for slugbuilder (default: "Always")
+DOCKER_BUILDER_IMAGE_PULL_POLICY                | the kubernetes [image pull policy][pull-policy] for dockerbuilder (default: "Always")
+DEIS_DEPLOY_BATCHES                             | the number of pods to bring up and take down sequentially during a scale (default: all)
+KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS | how many seconds kubernetes waits for a pod to finish work after a SIGTERM before sending SIGKILL (default: 30)
+
+## Customizing the Database
+
+The following environment variables are tunable for the [Database][] component:
+
+Setting           | Description
+----------------- | ---------------------------------
+BACKUP_FREQUENCY  | how often the database should perform a base backup (default: "12h")
+BACKUPS_TO_RETAIN | number of base backups the backing store should retain (default: 5)
+
+## Customizing Fluentd
+
+The following environment variables are tunable for [Fluentd][logger]:
+
+Setting           | Description
+----------------- | ---------------------------------
+SYSLOG_HOST_1     | The hostname of a remote syslog endpoint for shipping logs
+SYSLOG_PORT_1     | The port of a remote syslog endpoint for shipping logs
+
+## Customizing the Logger
+
+The following environment variables are tunable for the [Logger][] component:
+
+Setting           | Description
+----------------- | ---------------------------------
+STORAGE_ADAPTER   | How to store logs that are sent to the logger (default: "memory")
+NUMBER_OF_LINES   | How many lines to store in the ring buffer (default: 1000)
+
+## Customizing Minio
```

Comment:
hmm health server host and port make sense, but if a user were to change MINIO_HOST and MINIO_PORT they'd break minio. That's why I omitted it from the other components as well. :)